README.md

```diff
@@ -113,6 +119,8 @@ command.
 
 The connector MUST be on the same VPC network with the GKE cluster and configured with a not-in-use CIDR range.
 
+If your GKE cluster has enabled control plane authorized networks you must add the serverless connector CIDR range as an authorized network.
```

Comment:
Why ? It runs in the VPC, AFAIK the VPC is authorized. 

I never tried this, but if you did and that's how it behaves - it may be worth opening a bug on the serverless connector docs.